client/src/app/barmen/new/barman-new.component.html

```diff
@@ -30,19 +30,18 @@ <h1>Ajouter un Barman</h1>
                     Surnom <strong>requis</strong>
                     </mat-error>
                 </mat-form-field>
-                <mat-form-field>
+                <!-- <mat-form-field>
```

Comment:
Why comment this part ?
Back should create a new barman with an username (and password also but this is not in the spec I think)

Reply:
I would implement it in an other PR. You are right, I'm going to restore this file.